docs/management/managing-indices.asciidoc

```diff
@@ -0,0 +1,22 @@
+[[managing-indices]]
+== Managing Indices
+
+include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+
```

Comment:
This isn't necessary. The shared attributes are already included in the index.asciidoc file. We generally try to set attributes in the index files *unless* it's necessary to set/change them at a deeper level to facilitate conditional includes and such.